src/erlcloud_aws.erl

```diff
@@ -8,8 +8,8 @@
          aws_request_xml4/6, aws_request_xml4/8,
          aws_region_from_host/1,
          aws_request_form/8,
-         aws_request_form_raw/8,
-         do_aws_request_form_raw/9,
+         aws_request_form_raw/9,
```

Comment:
if only people were using this directly this might be a breaking change

Reply:
I can add forms of these for compatibility, hateful as that is.